src/BPEssentials/FileChecker.cs

```diff
@@ -20,7 +20,7 @@ public static async Task CheckFiles()
             }
             foreach (var file in RequiredFilesDictionary)
             {
-                if (File.Exists(file.Value)) continue;
+                if (File.Exists(file.Value)) { continue; }
```

Comment:
Please put `{` and `}` on new lines. It doesn't look as good, just makes it clearer to read.

```suggestion
                if (File.Exists(file.Value))
                {
                    continue;
                }
```